lms/envs/production.py

```diff
@@ -424,6 +425,9 @@
     CORS_ORIGIN_WHITELIST = ENV_TOKENS.get('CORS_ORIGIN_WHITELIST', ())
     CORS_ORIGIN_ALLOW_ALL = ENV_TOKENS.get('CORS_ORIGIN_ALLOW_ALL', False)
     CORS_ALLOW_INSECURE = ENV_TOKENS.get('CORS_ALLOW_INSECURE', False)
+    CORS_ALLOW_HEADERS = corsheaders_default_headers + (
```

Comment:
Why was this necessary when there's a `from .common import *` statement?

Reply:
I think since these variables are declared inside an `if` block, they are not importable from another module.